### ISSUE TITLE
Search results of dropdown component should be read by screenreaders

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/nextcloud-vue#2762]
This is an accessibility issue.

When the user starts typing in a search field some results are shown immediately. To follow this success criterion screen reader users must be notified that there are search results, and how many there are, the status of the search. But it’s important that this information is not conveyed while the user is still typing, as this can be very disturbing.

![image](https://user-images.githubusercontent.com/6078378/173393505-d99e6630-1fe1-45d0-b4b4-a1994cd7a4e0.png)

While searching for something, the search results change dynamically, but screen reader users aren’t notified that there are search results.

ToDo: evaluate the right implementation for all accessibility issues regarding this topic.

Suggestion for implementation: This can be done by adding aria-live=“polite” to the text.

**Comment by PVince81** [comment]
- [ ] also do this for the emoji picker, if not done already (to be verified)
